net/ddns-scripts/files/dynamic_dns_updater.sh

```diff
@@ -240,6 +240,15 @@ esac
 # without lookup host and possibly other required options we can do nothing for you
 [ -z "$lookup_host" ] && write_log 14 "Service section not configured correctly! Missing 'lookup_host'"
 
+# verify validity of variables
+[ -n "$lookup_host" ] && sanitize_variable lookup_host "$DNS_CHARSET" ""
+[ -n "$dns_server" ] && sanitize_variable dns_server "$DNS_CHARSET" ""
+[ -n "$domain" ] && sanitize_variable domain "$DNS_CHARSET" ""
```

Comment:
Please note that for Amazon Route 53 this is not a domain, but a zone ID, something like "Z2JRJ6QLK5O5DY". In this particular case, it does fit IN the DNS charset, but have you checked all plugins whether they reuse this field for something which is not a domain name?

Reply:
Good point! Looks like only the godaddy updater needs an extra character:

   option domain   - "yourdomain.TLD" to update or "hostname@yourdomain.TLD"

Will update the patch and add the '@' to the DNS_CHARSET.